### PR TITLE
[TASK] Streamline constant ordering

### PR DESCRIPTION
**Pull request by JPDesign**
# Pull Request

## Related Issues

* Closes #
* Fixes # [bug]
* Resolves # [feature/question]

## Prerequisites

* [ ] Changes have been tested on TYPO3 v8.7 LTS
* [ ] Changes have been tested on TYPO3 v9.5 LTS
* [ ] Changes have been tested on TYPO3 dev-master
* [ ] Changes have been tested on PHP 7.0.x
* [ ] Changes have been tested on PHP 7.1.x
* [ ] Changes have been tested on PHP 7.2.x
* [ ] Changes have been checked for CGL compliance `php-cs-fixer fix`

## Description

Do not overwrite 005_relatedVideos with 005_modestbranding.

## Steps to Validate 

1. [First Step]
2. [Second Step]
3. [and so on...]

### CODE REVIEW

**Comment by gilbertsoft** [comment]
Thanks for your PR. Well, actually this is not a bug but a typo. Your change will not fix a wrong behavior but the ordering of the constants in the constant editor. There are also similar problems which should also be addressed with this patch, can you please check everything and add more commits.

File: Configuration/TypoScript/ContentElement/constants.typoscript [hunk 1]
```diff
@@ -66,7 +66,7 @@ plugin.bootstrap_package_contentelements {
             showinfo = 0
             # cat=bootstrap package: content elements/131_contentelement/005_relatedVideos; type=boolean; label=Related Videos
```
Comment (gilbertsoft) [inline Configuration/TypoScript/ContentElement/constants.typoscript:67]: Number should be unique, 006 on this line

File: Configuration/TypoScript/ContentElement/constants.typoscript [hunk 2]
```diff
@@ -66,7 +66,7 @@ plugin.bootstrap_package_contentelements {
...
             relatedVideos = 0
-            # cat=bootstrap package: content elements/131_contentelement/005_relatedVideos; type=boolean; label=Modest Branding
+            # cat=bootstrap package: content elements/131_contentelement/005_modestbranding; type=boolean; label=Modest Branding
```
Comment (gilbertsoft) [inline Configuration/TypoScript/ContentElement/constants.typoscript:69]: Number should be 007 on this line

**Review by gilbertsoft — CHANGES_REQUESTED** [review]

**Comment by benjaminkott** [comment]
seems to be resolved with https://github.com/benjaminkott/bootstrap_package/commit/6e8c4a59a8ad934810b8cfab35824e163af93cfd

**Pull request closed without merging**